src/client/thirdParty/Amplitude.tsx

```diff
@@ -36,33 +37,51 @@ type AmplitudeProps = {
   apiKey: string;
 };
 
-export const Amplitude: FunctionComponent<AmplitudeProps> = ({ apiKey }) => {
+function initAmplitude(apiKey: string) {
+  amplitude.init(apiKey, {
+    defaultTracking: {
+      attribution: false,
+      sessions: false,
+      formInteractions: false,
+      fileDownloads: false,
+    },
+    serverUrl: AMPLITUDE_INGRESS_PROXY,
+  });
+}
+
+function usePlaygroundSignalsWrapper() {
   usePlaygroundSignals({
     onServerApiSuccess: (event) => {
       const visitorId = event.products.identification?.data?.visitorId;
       const botDetected = event?.products?.botd?.data?.bot?.result === 'bad' ? 'True' : 'False';
 
       amplitude.add(demoPageViewedEventPropertiesEnrichment(botDetected));
-      amplitude.init(apiKey, {
-        defaultTracking: {
-          pageViews: {
-            eventType: EVENT_TYPE,
```

Comment:
You cannot remove EVENT_TYPE from default tracking here, this completely breaks the original implementation of default page view tracking.